WalletWasabi.Backend/wwwroot/faq.html

```diff
@@ -153,13 +153,13 @@ <h1 class="text-center title-lined mb-5">FAQ</h1>
                                 WHAT IS A “COINJOIN”?
                             </label>
                             <div class="card-content">
-                                <p>Coinjoin is a collaborative transaction between multiple peers.</p>
+                                <p>A coinjoin is a collaborative transaction between multiple peers.</p>
```

Comment:
Are you sure about this?

`A coinjoin is a coll...` IMO this has one more 'a' than necessary.

Reply:
I think it is, as here coinjoin is a _noun_ (a coinjoin) and not a verb (to coinjoin)
ping expert @belliusz